raiden/tests/benchmark/profile_transfer.py

```diff
@@ -183,21 +191,37 @@ def profile_transfer(num_nodes=10, channels_per_node=2):
         for position in range(num_nodes)
     ]
 
-    BlockChainServiceMock.reset()
     blockchain_services = list()
+    tester = tester_state(
```

Comment:
This code:

```python
tester = tester_state(
        private_keys[0],
        private_keys,
        tester_blockgas_limit(),
    )
    nettingchannel_library_address = tester_nettingchannel_library_address(
        tester_state,
    )
    channelmanager_library_address = tester_channelmanager_library_address(
        tester_state,
        nettingchannel_library_address,
    )
    registry_address = tester_registry_address(
        tester_state,
        channelmanager_library_address,
    )
```

is duplicated here and in `raiden/tests/benchmark/speed.py` and `raiden/tests/benchmark/speed_transfer.py`

I know we would not save too many lines, but can you abstract it into a `tester_and_contracts_init()` function or something with a similar name?